src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/EventHandlerModifier/Sources/InvalidHandlerModifier_Expected.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using PX.Data;
+
+namespace Acuminator.Tests.Sources
+{
+	public class MyExtension : ParentExtension
+	{
+		protected virtual void HandleCacheAttachedDiscDate(Events.CacheAttached<PX.Objects.AR.ARInvoice.discDate> e) // compiler doesn't identify polymorphism, wrong modifier will be suggested.
```

Comment:
Could you please clarify what you meant here with this comment:
```cs
// compiler doesn't identify polymorphism, wrong modifier will be suggested.
```

Reply:
Done. That comment was confusing. I removed it. Thank you.